test/test_inverse.py

```diff
@@ -125,3 +128,27 @@ def test_NeumannInverseLinearOperator_toy():
 
     report_nonclose(inv_neumann, inv_ground_truth, rtol=1e-3, atol=1e-5)
     report_nonclose(inv_neumann @ y, inv_ground_truth @ y, rtol=1e-3, atol=1e-5)
+
+
+@mark.parametrize("cache", [True, False], ids=["cached", "uncached"])
+def test_KFAC_inverse_damped_matvec(case, cache: bool, delta: float = 1e-2):
+    """Test matrix-vector multiplication by an inverse damped KFAC approximation."""
+    model_func, loss_func, params, data = case
+
+    loss_average = "batch" if loss_func.reduction == "mean" else None
+    KFAC = KFACLinearOperator(
+        model_func, loss_func, params, data, loss_average=loss_average
+    )
+    KFAC._compute_kfac()
+    # add damping
+    for aaT, ggT in zip(
+        KFAC._input_covariances.values(), KFAC._gradient_covariances.values()
+    ):
+        aaT += delta * torch.eye(aaT.shape[0])
+        ggT += delta * torch.eye(ggT.shape[0])
```

Comment:
Nit: Could use `add_(eye(...), alpha=delta)`.

Reply:
Nit: I think this should crash on GPU because  the `eye` lives on the incorrect device.

Reply:
Question: is the tests in this file even run on multiple devices? I'm still sometimes confused by the use of `case`.

Reply:
If your local computer has a GPU, the tests will be run on both CPU and GPU:
- `case` fixture [loads](https://github.com/f-dangel/curvlinops/blob/ae0d25e04ecd606b32b275fd2f864dcb96ef6bda/test/conftest.py#L28-L29) model and loss function to the device